Source/WebSocket/hcwebsocket.cpp

```diff
@@ -240,38 +240,140 @@ try
 }
 CATCH_RETURN()
 
-HC_RESULT Internal_HCWebSocketConnect(
-    _In_z_ PCSTR uri,
+HC_API HC_RESULT HC_CALLING_CONV
+HCGlobalGetWebSocketFunctions(
+    _Out_ HC_WEBSOCKET_CONNECT_FUNC* websocketConnectFunc,
```

Comment:
out of curiosity why is setting/getting these three delegates separate from setting m_websocketMessageFunc and m_websocketCloseEventFunc?